api/backend/articles/schemas.py

```diff
@@ -0,0 +1,24 @@
+from marshmallow import Schema, fields
+import toastedmarshmallow
+
+# This schema is used to display article data
+class ArticleFormSchema(Schema):
+    title = fields.Str(required=True)
+    cover_image = fields.Str(required=False)
+    content = fields.Str(required=False)
+
+# This schema is used to display article data
+class ArticleSchema(Schema):
+    id = fields.Int(required=True)
+
+    class Meta:
+        # Fields to show when sending data
+        fields = ("id","title", "cover_image", "content","date_published","likes","is_published",)
+        ordered = True
+
+article_form_schema = ArticleFormSchema()
```

Comment:
Declare the schemas first, then the .Jit  schema
ex:
article_schema
articles_schema
article_schema.jit
article_schemas.jit